bm_scrapy/extensions.py

```diff
@@ -48,15 +48,16 @@ def from_crawler(cls, crawler):
     def item_scraped(self, item):
         item = self.exporter.export_item(item)
         data = {
-            "jid": os.getenv("BM_SPIDER_JOB"),
+            "jid": os.getenv("BM_COLLECTION"),
```

Comment:
I'm not sure these are the best suited variable names here 🤔 . It can be confusing why `jid` is equal to `BM_COLLECTION`. I think changing names to improve readability and coherence is needed (although not necessarily in this ticket).